### ISSUE TITLE
[Feature request] restoring some console specific data such as hwcal

### ISSUE THREAD

**Issue by monkeyman4412** [d0k3/GodMode9#462]
So restoring hwcal0.dat and hwcal1.dat may be possible. https://www.3dbrew.org/wiki/I2C_Registers
If you look under device 14, it states that hwcal0-1 actually is stored in the EEPROM or i2c... (not sure I understand it correctly) of the 3ds and the nand version is a copy of the EEPROM hwcal. Then you have the fact it's also possible to dump the original secureinfo and localfriendcode seed. as this tool https://github.com/joel16/3DS-Recovery-Tool is cable of doing so. Would it be possible to implement these features into godmode9, or at least the hwcal restoration?

**Comment by d0k3** [comment]
Quoting profi200 for the `hwcal0.dat` and `hwcal1.dat` files here:
> 12:55 <@profi200> https://github.com/d0k3/GodMode9/issues/462 That EEPROM doesn't exist on retail units unfortunately.

As for `SecureInfo_X`, 3DS Recovery Tool simply pulls that from CTRNAND:
https://github.com/joel16/3DS-Recovery-Tool/blob/master/source/main.c#L550-L556

`LocalFriendCodeSeed_X` is also pulled from CTRNAND, and as a last resort the output from some userland function (not accessible in GM9) is used:
https://github.com/joel16/3DS-Recovery-Tool/blob/master/source/main.c#L535-L549

Tl;dr: Simply copy these files as any other regular file in GM9. If you want to, write a script to simplify it. GM9 can't restore these from anywhere else.
